### PR TITLE
[SMALL FIX] Set some details of user as empty in default. 

### PR DESCRIPTION
**Pull request by Devesh21700Kumar**
1). Set some details of user as empty in default
2). Remove wishlist from update user dto and fixed tests

### CODE REVIEW

**Review by Devesh21700Kumar — COMMENTED** [review]
@Abhishek-kumar09 some minor changes and fixes for better use case from the client side.
I think this is ready for merge

File: src/modules/user/schema/user.schema.ts [hunk 1]
```diff
@@ -6,28 +6,28 @@ export type UserDocument = User & Document;
 
 @Schema({ timestamps: true })
 export class User {
-  @Prop()
+  @Prop({ default: '' })
```
Comment (Abhishek-kumar09) [inline src/modules/user/schema/user.schema.ts:9]: Why these changes? The create user dto is already imposing the the requirements of filling those fields.

Comment (Devesh21700Kumar) [reply src/modules/user/schema/user.schema.ts:9]: And create dto is not needed but update dto is needed
